src/main/java/mate/academy/spring/dto/request/MovieRequestDto.java

```diff
@@ -1,7 +1,16 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
 public class MovieRequestDto {
+    @NotNull
+    @NotEmpty
```

Comment:
you don't need notnull if you use notempty as it checks if the value is not null and size > 0. Please fix the same issue in all places